1-exercises/C-undefined-properties/exercise.js

```diff
@@ -14,24 +14,24 @@ let car = {
   yearsOld: 8,
 };
 
-console.log(car["colour"]);
+console.log(car["colour"]); // we did,t define color in car object.
 
 // Example 2
 function sayHelloToUser(user) {
-  console.log(`Hello ${user.firstName}`);
+  console.log(`Hello ${user.firstName}`); // we don't have .firstname property for user , we have .name .
 }
 
 let user = {
-  name: "Mira"
+  name: "Mira",
 };
 
 sayHelloToUser(user);
 
 // Example 3
 let myPet = {
   animal: "Cat",
-  getName: function() {
-    "My pet's name is Fluffy";
+  getName: function () {
+    "My pet's name is Fluffy"; // Function defination isn't correct , } missing.
   },
```

Comment:
not quite correct, undefined is returned because the function `getName()` does not return anything, it just defines a string.